api/src/certification/complementary-certification/domain/models/BadgeToAttach.js

```diff
@@ -44,6 +47,7 @@ class BadgeToAttach {
       stickerUrl,
       detachedAt: null,
       createdBy: userId,
+      minimumEarnedPix: minimumEarnedPix ?? 0,
```

Comment:
A + sa place dans son constructeur, c'est un invariant
```suggestion
      minimumEarnedPix: minimumEarnedPix,
```